src/Navigation/WPF/Navigation.WPF/NavigationServiceExtensions.cs

```diff
@@ -256,9 +274,8 @@ public static class NavigationServiceExtensions
         public static RegistrationInfo OpenInNewWindow(
             this RegistrationInfo registrationInfo)
         {
-            registrationInfo.OpenInNewWindow = true;
-
-            return registrationInfo;
+            throw new NotImplementedException(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Implement this method or throw 'NotSupportedException' instead.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5575746)